astro/src/content/json/api-keys/put-response-body.json

```diff
@@ -1,5 +1,6 @@
 {
   "apiKey": {
+    "expirationInstant": 1734824925683,
```

Comment:
Can we make the date 11 July 2029 (which is the date the resistance took out skynet: https://terminator.fandom.com/wiki/2029 )

1880046000000

Or another fun date to be an easter egg? I think Silicon Valley all takes place in the past, unfortunately.

Reply:
I like this idea. @lyleschemmerling knows I don't need a lot of excuses to make Terminator references. I think I'll make the post the date of Judgement Day and then our PUT request can update it to the day the war on the machines ends.